src/Photocard/Photocard.scss

```diff
@@ -0,0 +1,20 @@
+.photocard{
+  border : solid 2px black;
+  display : flex;
+  flex-direction : column;
+  align-items: center;
+  width : 300px;
+  height : 450px;
+  
+  .contentImage{
+    //border : solid 1px crimson;
```

Comment:
We should try to avoid having commented out code hanging around.
If we aren't using the code, its generally better to just delete it.

Reply:
We have removed commented out code